### ISSUE TITLE
Build problem

### ISSUE THREAD

**Issue by fmelodia** [wuusn/miyano#6]
When I try to build the webpage, I have an error : 
```
francois@Mini:~/essai|⇒  miyano build
Traceback (most recent call last):
	22: from /usr/local/bin/miyano:23:in `<main>'
	21: from /usr/local/bin/miyano:23:in `load'
	20: from /usr/local/lib/ruby/gems/2.5.0/gems/miyano-0.3.2/exe/miyano:18:in `<top (required)>'
	19: from /usr/local/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/base.rb:466:in `start'
	18: from /usr/local/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor.rb:387:in `dispatch'
	17: from /usr/local/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:126:in `invoke_command'
	16: from /usr/local/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/command.rb:27:in `run'
	15: from /usr/local/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor.rb:40:in `block in register'
	14: from /usr/local/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:115:in `invoke'
	13: from /usr/local/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/group.rb:232:in `dispatch'
	12: from /usr/local/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:133:in `invoke_all'
	11: from /usr/local/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:133:in `map'
	10: from /usr/local/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:133:in `each'
	 9: from /usr/local/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:133:in `block in invoke_all'
	 8: from /usr/local/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:126:in `invoke_command'
	 7: from /usr/local/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/command.rb:27:in `run'
	 6: from /usr/local/lib/ruby/gems/2.5.0/gems/miyano-0.3.2/lib/miyano/cli/build.rb:22:in `build'
	 5: from /usr/local/lib/ruby/gems/2.5.0/gems/miyano-0.3.2/lib/miyano/builder.rb:16:in `build_the_world'
	 4: from /usr/local/lib/ruby/gems/2.5.0/gems/miyano-0.3.2/lib/miyano/builder.rb:16:in `each'
	 3: from /usr/local/lib/ruby/gems/2.5.0/gems/miyano-0.3.2/lib/miyano/builder.rb:17:in `block in build_the_world'
	 2: from /usr/local/lib/ruby/gems/2.5.0/gems/miyano-0.3.2/lib/miyano/build/bear.rb:11:in `build_bear'
	 1: from /usr/local/lib/ruby/gems/2.5.0/gems/miyano-0.3.2/lib/miyano/build/bear.rb:11:in `each'
/usr/local/lib/ruby/gems/2.5.0/gems/miyano-0.3.2/lib/miyano/build/bear.rb:29:in `block in build_bear': undefined method `get_input_stream' for nil:NilClass (NoMethodError) 
```
It happened only when I use .bearnote with picture attached

**Comment by wuusn** [comment]
hi @fmelodia ,
Thanks for your try.

The information you posted here is still not enough to find the cause.
Could you attach your `bearnote` file here with the least information(for your content privacy) that will cause this problem.
You can also sent it to my e-mail (hi@wuyu.xin).

**Comment by fmelodia** [comment]
Ok (sorry for the delay, it's family time here ^^)

So, first : new blog
![1](https://user-images.githubusercontent.com/4469688/39943114-02da8cae-5562-11e8-983a-0adc0a95e3bc.jpeg)

First try : only text
![2](https://user-images.githubusercontent.com/4469688/39943176-388108e2-5562-11e8-9d15-fe66f1ef2f9e.jpeg)

Second try : picture attached
![3](https://user-images.githubusercontent.com/4469688/39943192-4b1ca40c-5562-11e8-8e86-a7c1b4842aae.jpeg)

I've joined the bearnote files (just renamed to .zip for the upload)

[First try.zip](https://github.com/wuusn/miyano/files/1996617/First.try.zip)
[Picture try.zip](https://github.com/wuusn/miyano/files/1996618/Picture.try.zip)

**Comment by wuusn** [comment]
hi @fmelodia ,

Sorry for the delay.
This issue was fixed on the latest version(0.3.3), please run `$ gem update`, 
then try again.

Thanks for your useful information.

**Comment by fmelodia** [comment]
Great ! It's working !
